.travis.yml

```diff
@@ -4,6 +4,7 @@
 # schema_dev gets run.
 ---
 sudo: false
+dist: precise
```

Comment:
Travis is changing the [default](https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming?utm_source=web&utm_medium=banner&&utm_campaign=trusty-default) linux for Trusty. When Travis tested through [my repository](https://travis-ci.org/iagopiimenta/schema_plus_core/jobs/256666738), it used Trusty, did not check why, but the mysql configuration did not work correctly. I've set the Need manually for now to solve this. His environment in Travis, for now, went up with the Precise.